### PR TITLE
Packaging.

### PR DESCRIPTION
**Pull request by Gozala**
This uses babel and bunch of other tools to generate minimized js bundles:

- browser/index
- service/history-worker
- about/settings/index

### CODE REVIEW

**Comment by Gozala** [comment]
@paulrouget can you please take a loot at this ? 
This makes load & render sync but I'm afraid initial one takes too long.

**Comment by Gozala** [comment]
This addresses #522 and #527 (to some degree) as with this js load and initial render becomes synchronous.

File: gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+var path = require('path');
+
+var settings = {
+  cache: {},
+  packageCache: {},
+  debug: true,
+  watch: false,
+  compression: null
+};
+
```
Comment (paulrouget) [inline gulpfile.js]: Are these settings good for a release build?

**Comment by paulrouget** [comment]
This looks good, but indeed, this slows down the first rendering quite a bit.

Let's keep it that way for now. We will figure prerendering in a future PR.

Feel free to merge.

File: gulpfile.js (hunk 1 above)
Comment (Gozala) [inline gulpfile.js]: > Are these settings good for a release build?

Yes, but that's an excellent question, I'll add comments to explain what those prefs are for.

**Pull request merged**